helm/ffc-workstream/values.yaml

```diff
@@ -0,0 +1,15 @@
+name: ffc-demo
+namespace: ffc-demo
+workstream: ffc-demo
+environment: production
+
+labels:
+  instance:
```

Comment:
`instance` isn't actually used anywhere so isn't a required value. The instance label is always set to `.Release.Name`.